lib/marked.js

```diff
@@ -730,6 +730,14 @@ InlineLexer.prototype.output = function(src) {
       } else if (this.inLink && /^<\/a>/i.test(cap[0])) {
         this.inLink = false;
       }
+      if (!this.inRawBlock
+          && /^<(pre|code|kbd|script)/i.test(cap[0])) {
```

Comment:
There should be a `(\s|>)` at the end to prevent matching other tags that start with these words

Reply:
Good one Tony; I've done so and expanded the tests to prove that.

> Loving your attention to detail here, much appreciated 👍